### PR TITLE
databricks

### PR DESCRIPTION
**Pull request by fivetran-jamie**
**Are you a current Fivetran customer?** 

fivetran made pr
**What change(s) does this PR introduce?** 

ensures databricks compatibility. no code changes were required in the transform package, but make sure https://github.com/fivetran/dbt_fivetran_utils/pull/98 is merged first 

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature https://github.com/fivetran/dbt_netsuite/issues/55
- [ ] No 

**How did you test the PR changes?** 

 

- [x] BuildKite  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [ ] BigQuery
- [ ] Redshift
- [ ] Snowflake
- [ ] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🧱 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,7 +80,15 @@ To use this dbt package, you must have At least either one Fivetran **Netsuite**
 - vendorcategory
 
 ### Database Compatibility
-This package is compatible with either a **BigQuery**, **Snowflake**, **Redshift**, or **PostgreSQL** destination.
+This package is compatible with either a **BigQuery**, **Snowflake**, **Redshift**, **PostgreSQL**, or **Databricks** destination.
```
Comment (fivetran-avinash) [inline README.md:83]: Same comment as the source package (Updating it to reflect a Part 1)

> ## Step 1: Prerequisites
> To use this dbt package, you must have the following:
> 
> - At least one Fivetran Netsuite connector syncing data into your destination.
> - A **BigQuery**, **Snowflake**, **Redshift**, **Databricks**, or **PostgreSQL** destination.

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -56,4 +56,18 @@ steps:
             - "CI_REDSHIFT_DBT_PASS"
             - "CI_REDSHIFT_DBT_USER"
     commands: |
-      bash .buildkite/scripts/run_models.sh redshift
\ No newline at end of file
+      bash .buildkite/scripts/run_models.sh redshift
+
+  - label: ":databricks: Run Tests - Databricks"
```
Comment (fivetran-avinash) [inline .buildkite/pipeline.yml:61]:
```suggestion
  - label: ":bricks: Run Tests - Databricks"
```

Don't think there's a :databricks: emoji =P

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# dbt_netsuite v0.7.1
+
```
Comment (fivetran-avinash) [inline CHANGELOG.md:2]:
```suggestion
## 🎉 Feature Updates 🎉
```

File: packages.yml [hunk 4]
```diff
@@ -1,3 +1,9 @@
 packages:
```
Comment (fivetran-avinash) [inline packages.yml:1]: Reminder to revert!

**Review by fivetran-avinash — CHANGES_REQUESTED** [review]
@fivetran-jamie Mostly good! Some changes but mostly syntax.

File: README.md (hunk 1 above)
Comment (fivetran-jamie) [reply README.md:83]: ah yeah so this step is formatted differently due to the Netsuite vs Netsuite2 section

File: .buildkite/pipeline.yml (hunk 2 above)
Comment (fivetran-jamie) [reply .buildkite/pipeline.yml:61]: i think there is one in buildkite! 
image

File: README.md (hunk 1 above)
Comment (fivetran-avinash) [reply README.md:83]: My bad.

**Review by fivetran-avinash — APPROVED** [review]
@fivetran-jamie LGTM!

**Pull request merged**